package.json

```diff
@@ -144,6 +144,9 @@
     "webpack-cli": "^3.2.3",
     "webpack-dev-server": "^3.2.1"
   },
+  "resolutions": {
+    "fsevents": "^2.0.0"
```

Comment:
eh?

Reply:
Something is on an old version that don't work with node 12, no issues upgrading that I could see